src/middlewared/middlewared/plugins/virt/disk.py

```diff
@@ -0,0 +1,178 @@
+import errno
+import os
+import re
+import shlex
+import subprocess
+
+from middlewared.api import api_method
+from middlewared.api.current import (
+    VirtDeviceImportDiskImageArgs, VirtDeviceImportDiskImageResult, VirtDeviceExportDiskImageArgs,
+    VirtDeviceExportDiskImageResult,
+)
+from middlewared.plugins.zfs_.utils import zvol_name_to_path
+from middlewared.service import CallError, job, Service
+
+
+# Valid Disk Formats we can export
+VALID_DISK_FORMATS = ['qcow2', 'qed', 'raw', 'vdi', 'vpc', 'vmdk']
```

Comment:
global variables should be immutable
```suggestion
VALID_DISK_FORMATS = ('qcow2', 'qed', 'raw', 'vdi', 'vpc', 'vmdk')
```